src/help.js

```diff
@@ -0,0 +1,29 @@
+import shell from 'shelljs';
+
+const badCommands = ['ShellString', 'cd', 'pushd', 'popd', 'dirs'];
```

Comment:
Done

Thanks! Ari
On Thu, May 5, 2016 at 5:11 PM, Levi Thomason <notifications@github.com> wrote:
In src/help.js [https://github.com/shelljs/shx/pull/49#discussion_r62274779] :

> @@ -0,0 +1,29 @@
> +import shell from 'shelljs';
> +
> +const badCommands = ['ShellString', 'cd', 'pushd', 'popd', 'dirs'];


There is a CMD_BLACKLIST already defined in shx.js . Suggest pulling that and the ERROR_CODES into a config.js or something so it can imported into both places.

Then you can filter based on the same list.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub [https://github.com/shelljs/shx/pull/49/files/c476547d608b00becee7cd1584e944c2d134c034#r62274779]